third_party/move/move-prover/boogie-backend/src/options.rs

```diff
@@ -22,7 +22,7 @@ const DEFAULT_BOOGIE_FLAGS: &[&str] = &[
 /// but not always. Setting the max version allows Prover to warn users for the higher version of
 /// boogie and z3 because those may be incompatible.
 const MIN_BOOGIE_VERSION: Option<&str> = Some("3.0.1.0");
-const MAX_BOOGIE_VERSION: Option<&str> = Some("3.0.9.0");
+const MAX_BOOGIE_VERSION: Option<&str> = Some("3.2.4.0");
```

Comment:
Set the version to 3.2.4 so that users who already installed 3.2.4 will not be bothered by the version error generated by CLI.